test/unit/ItemsContainer.test.js

```diff
@@ -4,10 +4,11 @@ import React from 'react';
 import { expect } from 'chai';
 import { shallow, mount } from 'enzyme';
 
-import ItemsContainer from './../../src/app/components/Item/ItemsContainer';
+import itemContainerExport from './../../src/app/components/Item/ItemsContainer';
```

Comment:
This called `itemsContainerModule` elsewhere

Reply:
I've renamed this to make it more consistent.